src/modules/cmdpal/Microsoft.CmdPal.UI/ShellPage.xaml.cs

```diff
@@ -44,5 +46,9 @@ public void Receive(NavigateBackMessage message)
         }
     }
 
-    public void Receive(NavigateToListMessage message) => RootFrame.Navigate(typeof(ListPage), message.ViewModel, _drillInNavigationTransitionInfo);
+    public void Receive(NavigateToListMessage message)
+    {
+        RootFrame.Navigate(typeof(ListPage), message.ViewModel, _slideRightTransition);
+        this.SearchBox.Focus(Microsoft.UI.Xaml.FocusState.Programmatic);
```

Comment:
```suggestion
        SearchBox.Focus(Microsoft.UI.Xaml.FocusState.Programmatic);
```

General convention these days is to not put this (as why type more), though I know the XAML templates include it for InitializeComponent by default (@niels9001 another thing for us to update).

I think there's an .editorconfig we can specify for that too?

Reply:
Ha you know, that was actually me just trying to kick intellisense into updating. I'll go pull it 😛